x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/application/LogsDBFeatureSetUsage.java

```diff
@@ -74,11 +89,12 @@ protected void innerXContent(XContentBuilder builder, Params params) throws IOEx
         builder.field("indices_with_synthetic_source", indicesWithSyntheticSource);
         builder.field("num_docs", numDocs);
         builder.field("size_in_bytes", sizeInBytes);
+        builder.field("has_custom_cutoff_date", hasCustomCutoffDate);
```

Comment:
Can you update the telemetry mappings after this PR?

Reply:
yes, I already have a pr: elastic/telemetry#4260